src/webfaf/templates/reports/item.html

```diff
@@ -64,6 +64,10 @@ <h3>Info</h3>
             {% if report.tainted %}
               <span class="label label-warning">Tainted</span>
             {% endif %}
+          </dd>
+          <dt> Component version </dt>
```

Comment:
It would be nice, if it would be clear it is the last affected version, so maybe reword it to `Last affected version`.